src/tls_stream.c

```diff
@@ -9,8 +9,8 @@
 
```

Comment:
Is the "tls_stream" file special in any way? I'd have expected it to be moved to the "streams" directory, as well.

Reply:
Will fix, I remember some conflict when rebasing, which I obviously mishandled.